core/src/main/java/org/apache/metamodel/query/Query.java

```diff
@@ -125,9 +125,35 @@ public Query select(String expression) {
         if ("*".equals(expression)) {
             return selectAll();
         }
-
-        SelectItem selectItem = findSelectItem(expression, true);
-        return select(selectItem);
+        
+        String[] possibleSelectItems = splitExpressionByComma(expression);
+        for(String possibleSelectItem : possibleSelectItems) {
+              SelectItem selectItem = findSelectItem(possibleSelectItem, true);
+              select(selectItem);
+        }
+        return this;
+    }
+   
+    /**
+     * If multiple columns or expressions are given in comma seperated string, this methods splits and generates
+     * string array of expressions representing single column.
+     * @param expression
+     * @return
+     */
+    public String[] splitExpressionByComma(String expression) {
```

Comment:
Maybe we can leverage some of the existing query parser code to do this. Please have a look at QueryParser class and whatever is related to select item parsing.